app/common/components/table.njk

```diff
@@ -11,10 +13,16 @@
     {% for row in params.data %}
       <div data-test-id="table-row-{{ loop.index0 }}" class="nhsuk-grid-row nhsuk-u-margin-1 nhsuk-u-padding-top-3 nhsuk-u-padding-bottom-3 bc-t-border-bottom--grey-3">
         {% for dataPoint in row %}
-          {% if params.columnInfo[loop.index0].link %}
-            <a class="nhsuk-u-padding-1 {{ params.columnClass }}" href="#">{{ dataPoint }}</a>
+          {% if dataPoint.href %}
+            <a class="nhsuk-u-padding-1 {{ params.columnClass }}" href="{{ dataPoint.href }}">{{ dataPoint.data }}</a>
```

Comment:
Just wondering if we can use our view-data-link component here?

Reply:
👍

Reply:
actually this adds in an extra div which is messing with the alignment

Reply:
😢 ok just leave as is for now then